### ISSUE TITLE
Remove Coverage extension settings

### ISSUE THREAD

**Issue by stdavis** [agrc/python#9]
These seem more like personal preferences than standards that we want to promote. Could they be easily moved to user settings?

https://github.com/agrc/python/blob/3b2391de75c5c50c4d629b1424c93fdf2248f9cf/.vscode/settings.json#L13-L18

**Comment by steveoh** [comment]
They definitely could. I found the defaults to be obnoxious so I thought these might be better defaults. Do you find these settings getting in your way?

**Comment by stdavis** [comment]
Yes, I just removed them. I wanted the coverage to show just in the gutters, not the entire line. The default colors didn't bother me.
![image](https://user-images.githubusercontent.com/1326248/91611208-ff110500-e937-11ea-8cff-f753339c58eb.png)

This is with the default for me.

I also wondered if we wanted to add `ryanluker.vscode-coverage-gutters` to the recommended extensions?

**Comment by steveoh** [comment]
Yes we can add that extension.

The reason I dislike the default gutter coverage is because it does not allow you to click in the gutter. Therefore breakpoints cannot be set for those debugging in vscode. https://github.com/ryanluker/vscode-coverage-gutters/issues/253
